pkg/cmd/admin.go

```diff
@@ -5,91 +5,90 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
-	"time"
 
 	"github.com/openshift/microshift/pkg/admin/data"
-	"github.com/openshift/microshift/pkg/admin/prerun"
-	"github.com/openshift/microshift/pkg/config"
+	"github.com/openshift/microshift/pkg/util"
 
 	"github.com/spf13/cobra"
 )
 
+func backupPathToStorageAndName(path string) (data.StoragePath, data.BackupName, error) {
```

Comment:
It would be good to have some unit tests for this function. 

What happens if the path is a relative path?

Reply:
It might be easier to force the path to an absolute path using `filepath.Abs()` instead of trying to deal with the different cases directly.

Reply:
`filepath.Clean()` could also be useful.

Reply:
Used `filepath.Clean()`, but skipped the `filepath.Abs()` because:
- it uses CWD, so values we expect in UT would become dynamic,
- it returns `(string, error)` so it's not as elegant so including it would mean: either adding it to 3 places (PreRun and 2* Run) before `backupPathToStorageAndName`) or using global vars (parse once in PreRun, use in Run) and I'm not a fan of any of them

I removed checking for `.` and `..` but had to add `/` because `filepath.{Dir,Base}("/")` returns `/`.

WDYT?

Reply:
It seems like the unit test could figure out what the expected values should be, but this is definitely an improvement.